### ISSUE TITLE
Capturing in the incorrect resolutions

### ISSUE THREAD

**Issue by lulzsun** [lulzsun/RePlays#39]
There is an issue with capturing games that display a splash screen window at the initial launch of the executable.

The initial approach we take before recording is to immediately retrieve the window handle of the game process that has just launched. If we retrieve the handle quick enough, it may end up being the splash screen window (or some semi-initialized window) and not the actual game window size.

Therefore, the output recording capture will be set at the resolution of the splash screen and not the actual game. This is not intended behavior.

Below are the following conflicted window handle class names that were recorded displaying this issue:
```
"SplashScreenWindow" - Tekken 7
"CoD Splash Screen" - Call of Duty®: Modern Warfare® II
"SDL_app" - Marauders
"EACLauncherWnd" - Squad
```

The occurrence is also common with splash screens that use Easy Anti Cheat.

Some proposed solutions:
- [x] Check the aspect ratio of the initial splash screen to see if it is a standard game resolution
- [x] Keep a dictionary/regex/string manipulation against conflicted window handle class names (partially implemented)
- [ ] Check periodically if the window resolution has changed

**Comment by brynjolf** [comment]
I run into this all the time to the point I made an AviSynth script just to crop the video to correct resolution. Let me know if you need help to reproduce it. 

It is quite annoying though

**Comment by lulzsun** [comment]
Are you currently on the nightly release? There were some changes to the recording backend (#109) that may have solved your issue. If you are on the nightly release, please provide the log files and mention any games that were giving you trouble, and I will take a look at it.

**Comment by brynjolf** [comment]
It seems to be working fine for me in the latest nightly. Not sure exactly which version fixed it for me since I been away during summer. Thank you anyhow :)

**Comment by lulzsun** [comment]
That's good to hear! Most of the recording issues have been resolved so far thanks to refactoring efforts (#109), and so I feel confident enough to be closing this issue for now. Thanks for reporting back!
